gulpfile.js

```diff
@@ -37,44 +40,43 @@ gulp.task('build', ['clean'], () => {
   const tsResult = tsProject.src().pipe(typescript(tsProject));
 
   return mergeStream(
-    tsResult.dts.pipe(gulp.dest('lib')),
-    tsResult.js.pipe(gulp.dest('lib'))
-  );
+      tsResult.dts.pipe(gulp.dest('lib')), tsResult.js.pipe(gulp.dest('lib')));
 });
 
-gulp.task('test', ['build'], () =>
-  gulp.src('test/**/*_test.js', {read: false})
-      .pipe(mocha({
-        ui: 'tdd',
-        reporter: 'spec',
-      }))
-);
+gulp.task(
+    'test',
+    ['build'],
+    () => gulp.src('test/**/*_test.js', {read: false}).pipe(mocha({
+      ui: 'tdd',
+      reporter: 'spec',
+    })));
 
-gulp.task('tslint', () =>
-  gulp.src('src/**/*.ts')
-    .pipe(tslint({
-      configuration: 'tslint.json',
-      formatter: 'verbose'
-    }))
-    .pipe(tslint.report()));
+gulp.task(
+    'tslint',
+    () =>
+        gulp.src('src/**/*.ts')
+            .pipe(tslint({configuration: 'tslint.json', formatter: 'verbose'}))
+            .pipe(tslint.report()));
 
-gulp.task('eslint', () =>
-  gulp.src('test/**/*.js')
-    .pipe(eslint())
-    .pipe(eslint.format())
-    .pipe(eslint.failAfterError()));
+gulp.task(
+    'eslint',
+    () => gulp.src('test/**/*.js')
+              .pipe(eslint({rules: {'comma-dangle': 2}}))
+              .pipe(eslint.format())
+              .pipe(eslint.failAfterError()));
 
-gulp.task('depcheck', () =>
-  depcheck(__dirname, {
-      // "@types/*" dependencies are type declarations that are automatically
-      // loaded by TypeScript during build. depcheck can't detect this
-      // so we ignore them here.
-      // "hydrolosis" is being incorrectly matched because it is no longer
-      // used directly in code. TODO: remove from this list once we remove
-      // as dependency.
-      ignoreMatches: ['@types/*', 'hydrolysis']
-    })
-    .then((result) => {
+gulp.task(
+    'depcheck',
+    () => depcheck(__dirname, {
+            // "@types/*" dependencies are type declarations that are
```

Comment:
I think we might want to stop using arrow function expressions for gulp task bodies. If we give them bodies and a return statement, clang might like it more.